src/rpc_client/src/meta_client.rs

```diff
@@ -883,8 +924,203 @@ impl GrpcMetaClient {
     // Max retry interval in ms for request to meta server.
     const REQUEST_RETRY_MAX_INTERVAL_MS: u64 = 5000;
 
+    async fn start_meta_election_monitor(
+        &self,
+        addr: &str,
```

Comment:
We'd better support configure multi-meta address for meta client in the future even though providing single address also works here. This might require some more changes in risedev, not emergency we can do it in anther PR.